saleor/order/utils.py

```diff
@@ -188,7 +189,7 @@ def move_order_line_to_group(line, target_group, quantity):
             delivery_group=target_group, product=line.product,
             product_name=line.product_name, product_sku=line.product_sku,
             quantity=quantity, unit_price_net=line.unit_price_net,
-            stock=line.stock,
+            stock=line.stock, variant=product_variant,
```

Comment:
What is the `product_variant` variable here? I don't see where it's defined. I think you meant `line.variant`.